### PR TITLE
chainAction

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] `chainAction` takes 1 param
- [x] logger uses function.name
- [x] no more name mangling in prod builds
- [x] kill old planned-billing code

### CODE REVIEW

File: shared/util/saga.tsx [hunk 1]
```diff
@@ -120,19 +130,60 @@ function* chainAction2Impl<Actions extends {readonly type: string}>(
...
+      action = undefined
+      if (toPut) {
+        const outActions: Array<TypedActions> = isArray(toPut) ? toPut : [toPut]
+        for (var out of outActions) {
+          if (out) {
+            yield Effects.put(out)
+          }
+        }
+      }
+      if (sl.isTagged) {
```
Comment (jakob223) [inline shared/util/saga.tsx]: Is the tag the thing you pass to the logger when you create it? if so, seems like this should always be tagged.

**Comment by chrisnojima** [comment]
somehow my webpack config changes got lost, i'll commit that back plus put in rn metro changes

File: shared/util/saga.tsx (hunk 1 above)
Comment (chrisnojima) [reply shared/util/saga.tsx]: this is only true if you call tag on the logger yourself

**Comment by chrisnojima** [comment]
@jakob223 config updated

**Pull request merged**